### DISCUSSION TITLE
Avocado Log Generation

### DISCUSSION THREAD

**Discussion by nithin2x** [avocado-framework/avocado#5280] [Q&A] [answered]
Hi,

Can anyone please help on how to get the job-results directory name of the currently running Test- inside that test?

**Comment by richtja** [comment, marked as answer]
Hi @nithin2x in `avocado-instrumented` tests you can use [logdir property](https://avocado-framework.readthedocs.io/en/latest/api/test/avocado.html?highlight=logdir#avocado.Test.logdir) which points to the path of the test logdir like:
```
~/avocado/job-results/{jobID}/test-results/{TestID}
```
